cfgov/legacy/templates/front/base_update.html

```diff
@@ -86,36 +109,43 @@
     {% endblock %}
 
     <!--[if lt IE 9]>
-    <link rel="stylesheet" href="{% static 'css/header.nonresponsive.css' %}"/>
-    <link rel="stylesheet" href="{% static 'css/footer.nonresponsive.css' %}"/>
+    <link rel="stylesheet" href="{{ STATIC_PREFIX }}css/header.nonresponsive.css"/>
+    <link rel="stylesheet" href="{{ STATIC_PREFIX }}css/footer.nonresponsive.css"/>
     <![endif]-->
 
     <!--[if gt IE 8]><!-->
-    <link rel="stylesheet" href="{% static 'css/header.css' %}"/>
-    <link rel="stylesheet" href="{% static 'css/footer.css' %}"/>
+    <link rel="stylesheet" href="{{ STATIC_PREFIX }}css/header.css"/>
+    <link rel="stylesheet" href="{{ STATIC_PREFIX }}css/footer.css"/>
     <!--<![endif]-->
 
+<!--
+    ============
+    HEAD SCRIPTS
+    ============
+    The following scripts must be included in the head. DO NOT ADD ANY MORE.
+    All other scripts should go before the closing body tag.
+    If you come across a script that makes a convincing case to be included in
+    the head, then file an issue or PR to discuss including it.
+-->
 
-    <!-- Feeds -->
-    <link rel="alternate" type="application/rss+xml" title="Consumer Financial Protection Bureau &raquo; Feed" href="/feed/" />
-    <link rel="alternate" type="application/rss+xml" title="Consumer Financial Protection Bureau &raquo; Comments Feed" href="/comments/feed/" />
-    {% block page_feeds %}
-    {% endblock %}
-
-    {% block analytics_js %}
-    <script src="{{ STATIC_PREFIX }}js/analytics.js{{ STATIC_VERSION }}"></script>
-
-    <!-- this is where we put our custom analytics -->
-    <script src="/static/nemo/_/js/analytics-global.js{{ STATIC_VERSION }}"></script>
```

Comment:
@bethannenc I'm removing `analytics-global.js` here from old pages that preceded our new Wagtail/jinja system. Is it okay to remove this? You said so in https://github.com/cfpb/cfgov-refresh/issues/496#issuecomment-201415389 for another project, but I'm not sure if it extends to the older stuff. For example, askcfpb would use this.